setup.py

```diff
@@ -25,4 +23,7 @@
         ],
     packages = ['isbn'],
     package_dir = {'isbn': 'isbn'},
-    )
+    autosemver={
+        'bugtracker_url': True,
```

Comment:
Maybe https://pypi.python.org/pypi/pyroma could help in catching other errors like these?